plugins/repository-azure/build.gradle

```diff
@@ -56,6 +56,23 @@ dependencies {
   api "io.netty:netty-transport-native-unix-common:${versions.netty}"
   implementation project(':modules:transport-netty4')
   api 'com.azure:azure-storage-blob:12.23.0'
+  api 'com.azure:azure-identity:1.11.4'
+  // Start of transitive dependencies for azure-identity
+  api 'com.microsoft.azure:msal4j-persistence-extension:1.2.0'
+  // Both azure-identity:1.11.4 and msal4j-persistence-extension:1.2.0 has compile dependency on different versions of jna-platform,
+  // selected the higher version which is 5.13.0
+  api 'net.java.dev.jna:jna-platform:5.13.0'
```

Comment:
`jna` and `jna-platform` are part of the same bundle and are of the same version: `${versions.jna}`

Reply:
@reta Thank you reta, I have made that change and rebase the branch